(dp1
S'output'
p2
S"<type 'exceptions.MemoryError'> "
p3
sS'layer'
p4
S'C:\\Users\\unificace\\unificace\\web2py\\applications\\init\\models\\db.py'
p5
sS'code'
p6
S'<code object <module> at 0000000002DDF7B0, file "C:\\Users\\unificace\\unificace\\web2py\\applications\\init\\models\\db.py", line 14>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File ".\\gluon\\restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "C:\\Users\\unificace\\unificace\\web2py\\applications\\init\\models\\db.py", line 992, in <module>\n    Field(\'file2\', \'upload\', uploadfield=\'file_data2\',label=T(\' \'),comment=T("Fichier CSV transform\xc3\xa9 et import\xc3\xa9"),represent=lambda value,row:A(\'Fichier CSV import\xc3\xa9\' if value else \'\',_href=URL(\'download\', args=value)),writable=False),\n  File "C:\\Users\\unificace\\unificace\\web2py\\gluon\\packages\\dal\\pydal\\objects.py", line 1637, in __init__\n    self.widget = widget\nMemoryError\n'
p11
s.